### PR TITLE
Style check review for minor issues

### PR DESCRIPTION
**Pull request by SparK-Cruz**
Full code check based on late review of #94

### CODE REVIEW

File: src/Models/BaseModel.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
      *
      * @param $attributes array
```
Comment (gpressutto5) [inline src/Models/BaseModel.php]: Wrong order

**Comment by gpressutto5** [comment]
@SparK-Cruz Is code coverage fixed?

**Comment by SparK-Cruz** [comment]
@gpressutto5 Check commit name ;)
Locally the report says yes, so codecov should have no trouble this time.

**Comment by gpressutto5** [comment]
@SparK-Cruz Commit name means nothing to me. I was waiting for codecov to say something, but since you said its ok, you can merge it.

**Pull request merged**